vmdb/db/migrate/20150501193927_default_provider_verify_ssl.rb

```diff
@@ -0,0 +1,9 @@
+class DefaultProviderVerifySsl < ActiveRecord::Migration
+  def up
+    change_column :providers, :verify_ssl, :integer, :null => false
```

Comment:
@Fryguy I explained why we don't want nil in here.
Are you suggesting we want to allow nil in this column?